### PR TITLE
Break getFeedback into separate feedback functions.

### PR DESCRIPTION
**Pull request by eyurko**

### CODE REVIEW

File: client/services/FeedbackGeneratorServiceSpec.js [hunk 1]
```diff
@@ -81,7 +81,9 @@ describe('FeedbackGeneratorService', function() {
       expect(paragraphs[1].isCodeParagraph()).toBe(true);
       expect(paragraphs[1].getContent()).toBe('ERROR MESSAGE');
     });
+  });
 
+  describe('_getTimeoutErrorTestFeedback', function() {
```
Comment (seanlip) [inline client/services/FeedbackGeneratorServiceSpec.js]: For consistency: no need "Test" in the name?

**Comment by seanlip** [comment]
Btw do you want to add a glob pattern to .gitignore for .DS_Store?

**Comment by eyurko** [comment]
I thought I did, but I'll do it tomorrow.

--
Eric Yurko

File: client/services/FeedbackGeneratorServiceSpec.js (hunk 1 above)
Comment (eyurko) [inline client/services/FeedbackGeneratorServiceSpec.js]: Whoops. Fixed.

**Comment by eyurko** [comment]
I think the .gitignore is already updated?

**Pull request merged**